CRM/Core/BAO/PrevNextCache.php

```diff
@@ -282,7 +282,7 @@ public static function retrieve($cacheKey, $join = NULL, $whereClause = NULL, $o
 
       if (!empty($select)) {
         $extraData = array();
-        foreach ($select as $dfield => $sfield) {
+        foreach ($select as $sfield) {
```

Comment:
I did a quick google & didn't find anything to support that. Mostly I found for vs foreach - e.g https://coderwall.com/p/il1tog/php-for-vs-foreach-benchmark - but in general most of what I found was php 5.3 or 5.4- I expect it changes each php version